src/utils/VecND.h

```diff
@@ -228,8 +227,8 @@ class VecND
   typename std::enable_if_t<NN==3, VecND<T,N>>
   cross(const VecND<T,N>& v) const
   { return {
-      y * v.z - z * v.y,
-      z * v.x - x * v.z,
+      y * v[2] - entries_ * v.y,
```

Comment:
Hey,

I think it should read `entries_[2]` at this point. 

Actually - I would like to stick to the `z` attribute of this class for a better readability of operations like this. 
Of course, it will lead to undefined behavior for `N < 3` - but I would but i would leave this burden to the user.

Or is there maybe some kine of `std::enable_if` pattern that can be applied to this class attributes?
Such that something like
```
VecND<int,2> foo;
foo.z = 3;
``` 
would result in a compilation error?

That would be great...

Anyway - thanks for your help :-)

Regards,
Flo

Reply:
Hello, thank you so much for your library. It's nice.

> Actually - I would like to stick to the `z` attribute of this class for a better readability of operations like this.

I made the change because somehow I wasn't able to build with `msvc` when the array script goes out of range. Maybe it's my fault. Would you like me to revert the change?